### PR TITLE
Enable parallel/async commit by default

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java [hunk 1]
```diff
@@ -82,9 +82,9 @@ public ConsensusCommitConfig(DatabaseConfig databaseConfig) {
             PARALLEL_EXECUTOR_COUNT,
             DEFAULT_PARALLEL_EXECUTOR_COUNT);
     parallelPreparationEnabled =
-        getBoolean(databaseConfig.getProperties(), PARALLEL_PREPARATION_ENABLED, false);
+        getBoolean(databaseConfig.getProperties(), PARALLEL_PREPARATION_ENABLED, true);
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java:85]: Enable parallel preparation by default.

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java [hunk 2]
```diff
@@ -82,9 +82,9 @@ public ConsensusCommitConfig(DatabaseConfig databaseConfig) {
...
     parallelCommitEnabled =
-        getBoolean(databaseConfig.getProperties(), PARALLEL_COMMIT_ENABLED, false);
+        getBoolean(databaseConfig.getProperties(), PARALLEL_COMMIT_ENABLED, true);
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java:87]: Enable parallel commit by default.

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java [hunk 3]
```diff
@@ -95,7 +95,7 @@ public ConsensusCommitConfig(DatabaseConfig databaseConfig) {
         getBoolean(
             databaseConfig.getProperties(), PARALLEL_ROLLBACK_ENABLED, parallelCommitEnabled);
 
-    asyncCommitEnabled = getBoolean(databaseConfig.getProperties(), ASYNC_COMMIT_ENABLED, false);
+    asyncCommitEnabled = getBoolean(databaseConfig.getProperties(), ASYNC_COMMIT_ENABLED, true);
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java:98]: Enable async commit by default.

File: core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java [hunk 4]
```diff
@@ -30,7 +30,7 @@ public class ConsensusCommitConfig {
   public static final String ASYNC_COMMIT_ENABLED = PREFIX + "async_commit.enabled";
   public static final String ASYNC_ROLLBACK_ENABLED = PREFIX + "async_rollback.enabled";
 
-  public static final int DEFAULT_PARALLEL_EXECUTOR_COUNT = 30;
+  public static final int DEFAULT_PARALLEL_EXECUTOR_COUNT = 128;
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java:33]: Also increased default parallel executor count.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**